src/images/coblox_bitcoincore.rs

```diff
@@ -93,6 +92,7 @@ pub struct BitcoinCoreImageArgs {
     pub accept_non_std_txn: Option<bool>,
     pub rest: bool,
     pub fallback_fee: Option<f64>,
+    pub address_type: String,
```

Comment:
Given that there is only a limited set of address-types, we could model this as an enum!

The valid options are:

- `legacy`
- `p2sh-segwit`
- `bech32`

Reply:
Great feedback! I've implemented it and commit the new change.